packages/gestalt/src/Toast.css

```diff
@@ -0,0 +1,7 @@
+.textColorOverrideWhite * {
+  color: var(--g-colorGray0) !important; /* stylelint-disable-line declaration-no-important */
```

Comment:
@dangerismycat when we roll it out, will we be able to avoid the `!important` and `*`? Both of these are anti patterns

Reply:
I don't think so, and that's the part I was unsure about. We need to be able to control the color of the `text` when it's a Node, and I couldn't think of a better solution.